lib/http/client.rb

```diff
@@ -24,7 +24,12 @@ def initialize(default_options = {})
       @state = :clean
     end
 
-    # Make an HTTP request
+    # Make an HTTP request.
+    # See HTTP::Options.defined_options for list of option keys.
```

Comment:
Please, replace this with:

```suggestion
    # @see Options
```